<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movix</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtresource.qrc">
    <normaloff>:/png/icon.png</normaloff>:/png/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="admin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adminPass">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>135</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>135</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="visible">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { color : #6ee6c7; } </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-60</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/png/logo.png);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/png/logo.png&quot;/&gt;&lt;img src=&quot;:/png/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>admin</zorder>
   <zorder>user</zorder>
   <zorder>adminPass</zorder>
   <zorder>loginBtn</zorder>
   <zorder>usernameEdit</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtresource.qrc"/>
 </resources>
 <connections/>
</ui>
